src/odin/commands/start.rs

```diff
@@ -1,25 +1,43 @@
 use crate::{
-  files::config::load_config,
+  files::config::{self, load_config},
   notifications::enums::{event_status::EventStatus, notification_event::NotificationEvent},
   server,
 };
 
 use log::{debug, error, info};
-use std::process::exit;
+use std::process::{exit, Child};
 
-pub fn invoke(dry_run: bool) {
+fn run_server(config: &config::ValheimArguments) -> Child {
+  server::start_daemonized(config)
+    .map_err(|err| format!("{}", err))
+    .and_then(|it| it.map_err(|err| format!("{}", err)))
+    .map_err(|err: String| {
+      error!(target: "commands_start", "Error: {}", err);
+      exit(1);
+    })
+    .expect("failed to daemonize server")
```

Comment:
i didn't see a repo error enum, but didn't want to a) introduce one or b) go `.unwrap()` crazy, so i opted to just use `String` as the `Err` variant to get sane messaging propagated up. not married to it (or any of this really) at all!